src/main/java/io/github/thebusybiscuit/slimefun4/api/events/SlimefunItemSpawnEvent.java

```diff
@@ -29,13 +33,33 @@ public class SlimefunItemSpawnEvent extends Event implements Cancellable {
     private ItemStack itemStack;
     private boolean cancelled;
     private final ItemSpawnReason itemSpawnReason;
+    private final Player player;
 
     @ParametersAreNonnullByDefault
     public SlimefunItemSpawnEvent(Location location, ItemStack itemStack, ItemSpawnReason itemSpawnReason) {
         this.location = location;
         this.itemStack = itemStack;
         this.itemSpawnReason = itemSpawnReason;
         this.cancelled = false;
+        this.player = null;
```

Comment:
You can replace this with a `this()` call

Reply:
No, that would require me adding a Player field to the other constructor (see your other comment on why that can't be)